sql/drop.go

```diff
@@ -0,0 +1,48 @@
+package sql
+
+import (
+	"fmt"
+
+	"github.com/cockroachdb/cockroach/keys"
+	"github.com/cockroachdb/cockroach/sql/parser"
+	"github.com/cockroachdb/cockroach/structured"
+)
+
+// DropTable drops a table.
+func (p *planner) DropTable(n *parser.DropTable) (planNode, error) {
```

Comment:
it only tests that the table is not reachable using the sql interface. The tests in sql/ should verify that the metadata has been properly modified. In this case, that means:
- both descriptors have been removed
- the data has been removed

The tests would pass just as well if the name->ID entry was deleted, but Drop does more.